File: src/Servidor/Transaccional/AccesoDatos_Fac/Dao/NHibernate/DaoEtiquetaNHibernate.vb
Imports NLog
Imports Diginsoft.Bolet.AccesoDatos.Contrato
Imports Trascend.Bolet.ObjetosComunes.Entidades
Imports Trascend.Bolet.AccesoDatos.Dao.NHibernate
Namespace Dao.NHibernate
    Public Class DaoEtiquetaNHibernate
        Inherits DaoBaseNHibernate(Of Etiqueta, String)
        Implements IDaoEtiqueta
        Private Shared logger As Logger = LogManager.GetCurrentClassLogger()
    End Class
End Namespace

File: src/Servidor/Transaccional/AccesoDatos/Dao/NHibernate/DaoEtiquetaNHibernate.cs
﻿using NLog;
using Trascend.Bolet.ObjetosComunes.Entidades;
using Trascend.Bolet.AccesoDatos.Contrato;

namespace Trascend.Bolet.AccesoDatos.Dao.NHibernate
{
    class DaoEtiquetaNHibernate : DaoBaseNHibernate<Etiqueta, string>, IDaoEtiqueta
    {
        private static Logger logger = LogManager.GetCurrentClassLogger();
    }
}
